rclc/include/rclc/executor.h

```diff
@@ -0,0 +1,387 @@
+// Copyright (c) 2019 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository https://github.com/ros2/rclc.
```

Comment:
I think by putting these two lines just below the rest of the license, the linters should be okay with that.

Reply:
Hmm, I don't get any errors from the linters. Which linters creates an error message here?